sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -5207,11 +5233,15 @@ public void testCaseFilteredAggregationWithGroupBy()
                         .setInterval(querySegmentSpec(Filtration.eternity()))
                         .setGranularity(Granularities.ALL)
                         .setDimensions(dimensions(new DefaultDimensionSpec("cnt", "d0", ColumnType.LONG)))
+                        .setVirtualColumns(
+                            expressionVirtualColumn(
+                                "v0",
+                                "case_searched((\"dim1\" != '1'),1,0)",
```

Comment:
this `CASE` statement was retained because the transformation to `COUNT` may change behaviour when  there are `0` input rows;
I've some ideas how to fix this - but didn't got to it yet